manifests/supervisorcluster/1.25/cns-csi.yaml

```diff
@@ -70,7 +70,7 @@ rules:
     resources: ["networkinterfaces"]
     verbs: ["get"]
   - apiGroups: ["cns.vmware.com"]
-    resources: ["cnsvolumeoperationrequests"]
+    resources: ["cnsvolumeoperationrequests", "storagepolicyusages"]
     verbs: ["create", "get", "list", "update", "delete"]
```

Comment:
do we need patch permission instead of update?

Reply:
We should use "patch" instead of "update" whenever possible.

Reply:
Done